runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -218,6 +217,74 @@ public void onWidgetLoaded(WidgetLoaded event)
 		}
 	}
 
+	@Subscribe
+	public void itemContainerChanged(ItemContainerChanged event)
+	{
+		final ItemContainer container = event.getItemContainer();
+
+		if (container == client.getItemContainer(InventoryID.INVENTORY))
+		{
+			final Item[] items = container.getItems();
+
+			long numberOfSupplyCrates = Arrays.stream(items)
+					.filter(item -> item.getId() == ItemID.SUPPLY_CRATE)
+					.count();
+
+			// We want to make sure that we actually opened the supply crate, not banked it
+			if (numberOfSupplyCrates != lastNumberOfSupplyCrates && inventorySnapshop!= null)
+			{
+				log.debug("Number of supply crates in inventory has changed from {} to {}", lastNumberOfSupplyCrates, numberOfSupplyCrates);
+
+				// we lost a crate by opening it
+				if (numberOfSupplyCrates < lastNumberOfSupplyCrates && !bankWidgetIsOpen())
+				{
+					List<Item> loot = itemDelta(inventorySnapshop, items);
+
+					final Collection<ItemStack> lootStacks = loot.stream()
+							.map(item -> new ItemStack(item.getId(), item.getQuantity()))
+							.collect(Collectors.toList());
+
+					if (!lootStacks.isEmpty())
+					{
+						final LootTrackerItem[] entries = buildEntries(stack(lootStacks));
+						SwingUtilities.invokeLater(() -> panel.add(WINTERTODT_EVENT_TYPE, -1, entries));
+					}
+				}
+
+				lastNumberOfSupplyCrates = numberOfSupplyCrates;
+			}
+
+			inventorySnapshop = items;
+		}
+	}
+
+	private List<Item> itemDelta(Item[] oldItems, Item[] newItems)
+	{
+		List<Item> deltaItems = new ArrayList<>();
+
+		for (int i = 0; i < newItems.length; i++)
+		{
```

Comment:
If the inventory items are in the order of their slots, this will trigger if you move items around.

Reply:
Every time there is an inventory update we update the snapshot, so unless you move items between the click on the box and the inventory update I don't think that's possible? I could be mistaken.

Reply:
Considering people can do X way switches, it's entirely possible